.circleci/config.yml

```diff
@@ -316,7 +323,9 @@ jobs:
             - C:\Users\circleci\go\pkg
             - C:\Go\pkg
   e2e-deprecated-windows:
-    executor: win/default
+    executor:
+      name: win/server-2022
+      version: current
```

Comment:
Why is this version different to the others? Could we use the same version for all the jobs?

Reply:
my bad